doc/avifdec.1.md

```diff
@@ -49,73 +47,47 @@ Output format can be either JPEG, PNG or YUV4MPEG2 (Y4M).
         - **libgav1**
 
 **-d**, **\--depth** _D_
-:   Output PNG depth.
-    Ignored when the output format is JPEG (always 8 bits per channel) or Y4M
-    (input depth is retained).
-
-    Possible values are:
-
-    :   - **8**
-        - **16**
+:   Output depth [8,16]. (PNG only; For y4m, depth is retained, and JPEG is always 8bpc).
```

Comment:
we have some inconsistency with how we note intervals and sets. How about `{8,16}` for a finite set of values and `(0..100)` (WebP notation in the command line) for an interval?

Reply:
I will leave this as is until we have a decision or for some future PR :)